migrations/sql/V2024.02.15.10.40__project_summary_facility_operator.sql

```diff
@@ -0,0 +1,30 @@
+-- add party for facility operator
+ALTER TABLE project_summary
+    ADD COLUMN IF NOT EXISTS facility_operator_guid UUID,
+    ADD COLUMN IF NOT EXISTS facility_type VARCHAR,
```

Comment:
Default character length here?

Reply:
Yeah I agree. I'll hound UX/Rebecca on this.